src/commands/site-list.php

```diff
@@ -39,120 +47,157 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
+				array( 'It works by searching for the term in the site url and if found,', '' ),
+				array( 'the site is excluded unless explicitly overridden.', '' ),
+				array( 'Term list:', '' ),
+			);
+			foreach ( $ignore as $term ) {
+				$filters_output[] = array( $term, '' );
+			}
+			$filters_output[] = array( 'The following sites are allowed to pass the above filtered terms and thus,', '' );
+			$filters_output[] = array( 'counted as live sites:', '' );
+			foreach ( $free_pass as $pass ) {
+				$filters_output[] = array( $pass, '' );
 			}
 
-			$final_site_list[] = array(
-				preg_replace( '/[^a-zA-Z0-9\s&!\/|\'#.()-:]/', '', $site->name ),
-				$site->URL,
-				$site->ID,
-				$server,
+			$summary_output = array(
+				array( 'REPORT SUMMARY', '' ),
+				array( 'Private sites', $this->count_sites( $audited_site_list, 'is_private', 4 ) ),
+				array( "'Coming Soon' sites", $this->count_sites( $audited_site_list, 'is_coming_soon', 5 ) ),
+				array( 'Multisite parent sites', $this->count_sites( $audited_site_list, 'is_parent', 6 ) ),
+				array( 'Multisite subsites', $this->count_sites( $audited_site_list, 'is_subsite', 6 ) ),
+				array( 'Domain only sites', $this->count_sites( $audited_site_list, 'is_domain_only', 7 ) ),
+				array( 'Atomic sites', $this->count_sites( $audited_site_list, 'Atomic', 8 ) ),
+				array( 'Pressable sites', $this->count_sites( $audited_site_list, 'Pressable', 8 ) ),
+				array( 'Simple sites', $this->count_sites( $audited_site_list, 'Simple', 8 ) ),
+				array( 'Other hosts', $this->count_sites( $audited_site_list, 'Other', 8 ) ),
+				array( 'PASSED sites', $this->count_sites( $audited_site_list, 'PASS', 9 ) ),
+				array( 'FAILED sites', $this->count_sites( $audited_site_list, 'FAIL', 9 ) ),
+				array( 'Total sites', count( $audited_site_list ) ),
+				array( 'AUDIT TYPE/FILTER', $audit_type ),
 			);
-		}
-
-		$site_table = new Table( $output );
-		$site_table->setStyle( 'box-double' );
-		$site_table->setHeaders( array( 'Site Name', 'Domain', 'Site ID', 'Host' ) );
+			foreach ( $filters_output as $line ) {
+				$output->writeln( "<info>{$line[0]}<info>" );
+			}
+			$output->writeln( "\n" );
 
-		$site_table->setRows( $final_site_list );
-		$site_table->render();
+			foreach ( $summary_output as $line ) {
+				$output->writeln( "<info>{$line[0]}: {$line[1]}<info>" );
+			}
 
-		$atomic_count    = $this->count_sites( $final_site_list, 'Atomic' );
-		$pressable_count = $this->count_sites( $final_site_list, 'Pressable' );
-		$other_count     = $this->count_sites( $final_site_list, 'Other' );
-		$simple_count    = $this->count_sites( $final_site_list, 'Simple' );
+			$summary_output  = array_merge( $filters_output, $summary_output );
+			$final_site_list = $audited_site_list;
+		} else {
+			$final_site_list = $this->filter_public_sites( $full_site_list );
+
+			$site_table = new Table( $output );
+			$site_table->setStyle( 'box-double' );
+			$table_header = array( 'Site Name', 'Domain', 'Site ID', 'Host' );
```

Comment:
If we use an assoc. array, maybe we could use its keys here as headers (could be parsed to improve their readability).

Reply:
Completed in https://github.com/Automattic/team51-cli/pull/159/commits/f7dcce423becc7731b6acd5c0c6b98dcaf7840c0